data/water/wetlands/index.html

```diff
@@ -10,40 +9,78 @@
 categories: []
 date: 2011-11-22 14:47:08 -0700
 Wetlands:
-  downloads:
-    - url: https://drive.google.com/file/d/0ByStJjVZ7c7mbnlUUTNWbWdOclU
-      label: 'Wetlands: File Geodatabase'
-    - url: https://drive.google.com/drive/u/0/folders/1mjTOCRivKo9g1vO_0xzlw4I1e3ihKjCf
-      label: 'Wetlands Metadata: File Geodatabase'
-    - url: https://drive.google.com/drive/u/0/folders/0ByStJjVZ7c7mNW9iXzJrQUYyV0U
-      label: 'Wetlands: Shapefile'
-    - url: https://drive.google.com/drive/u/0/folders/1mjTOCRivKo9g1vO_0xzlw4I1e3ihKjCf
-      label: 'Wetlands Metadata: Shapefile'
+  hub:
+    name: Utah Wetlands
+    item_id: dbc6a99d04944dab9f23bb6286843a1d
   updates:
-    - Wetlands will be updated approximately twice per year when NWI releases new updates. The current version is up-to-date as of June 18, 2019.
+    - Wetlands will be updated approximately twice per year when NWI releases new updates. The current version is up-to-date as of June 2020.
+Wetlands Mapping Projects:
```

Comment:
**issue**: these cannot contain spaces. please fix the others and the references to them as well.

```suggestion
WetlandsMappingProjects:
```